docs/sparkr.md

```diff
@@ -210,6 +210,43 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Model Formulae
+
```

Comment:
Could you also update the "Overview" section at the top to say something like `SparkR also supports distributed machine learning using MLlib` etc.